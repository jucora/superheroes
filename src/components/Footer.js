import React from 'react';

function Footer() {
  return (
    <div className="footerContainer">
      <footer>
        <p>Julian Belmonte copyright 2020</p>
      </footer>
    </div>
  );
}

export default Footer;
