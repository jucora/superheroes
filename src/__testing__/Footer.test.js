import React from "react";
import { render, cleanup } from "@testing-library/react";
import Footer from "../components/Footer";

afterEach(cleanup);

it("should take a snapshot", () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment(<Footer />)).toMatchSnapshot();
});
