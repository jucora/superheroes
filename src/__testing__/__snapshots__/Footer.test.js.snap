// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="footerContainer"
  >
    <footer>
      <p>
        Julian Belmonte copyright 2020
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
